ui/src/features/Editor/hooks.ts

```diff
@@ -355,6 +356,29 @@ export default ({
     handleWorkflowClose,
   });
 
+  const handleNodeDisable = useCallback(
```

Comment:
Pretty sure we want this with the rest of the yjs code or else we start mixing domains again